src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -157,6 +220,97 @@ export default class ExtensionRequest extends Vue {
     const data = { ...this.data, agmYear: value }
     this.emitData(data)
   }
+
+  get extensionExpiryMax (): string {
+    if (this.isFirstAgm) {
+      return DateUtilities.addMonthsToDate(30, DateUtilities.dateToYyyyMmDd(this.data.incorporationDate))
```

Comment:
In general, please add as many comments as possible to explain the business logic.

Here, please explain how we got to +30 months.

Ditto on line 234 below.

Reply:
Awesome! Now I get it. Super!